libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -17,7 +16,8 @@ describe('ItemComponent', () => {
   let spectator: SpectatorHost<ItemComponent>;
 
   const createHost = createHostFactory({
-    imports: [MockModule(IonicModule), IconModule],
+    imports: [IconModule],
+    providers: [provideIonicAngular({ mode: 'ios', _testing: true })],
```

Comment:
Consider using the testhelper here
```suggestion
    providers: [TestHelper.ionicModuleForTest],
```